xsuite/src/cli/buildReproducibleAction.ts

```diff
@@ -0,0 +1,205 @@
+import { PathLike, existsSync, mkdirSync, readdirSync } from "fs";
+import path from "path";
+import { Command } from "commander";
+import { log } from "../_stdio";
+import {
+  defaultIgnore,
+  findBuildableContractDirs,
+  getGid,
+  getUid,
+} from "./buildUtils";
+import {
+  defaultReproducibleDockerImage,
+  logAndRunCommand,
+  logError,
+  promptUserWithRetry,
+} from "./helpers";
+
+export const registerBuildReproducibleCmd = (cmd: Command) => {
+  cmd
+    .command("build-reproducible")
```

Comment:
It would be great to have a way to specify the Rust target folder in order to reuse the same target folder between reproducible builds and lower the reproducible build time. I need to discuss with the team to see if it would be feasible.